packages/components/src/Card/Card.spec.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import 'jest-styled-components'
+
+import Card from '.'
+
+describe('Card', () => {
+  test('default', () => {
+    const tree = renderer
+      .create(
+        <Card
+          data={{
+            title: 'Card story',
+            editors: [{ firstName: 'Story Writer' }],
+            start: Date.now(),
```

Comment:
`Date.now()` won't work nicely with snapshots. Replace with hardcoded value.

Reply:
Good catch thank you